exporter/exporterparser/zipkin.go

```diff
@@ -40,6 +40,7 @@ type ZipkinConfig struct {
 	ServiceName      string         `yaml:"service_name,omitempty"`
 	Endpoint         string         `yaml:"endpoint,omitempty"`
 	LocalEndpointURI string         `yaml:"local_endpoint,omitempty"`
+	BatchSize        int            `yaml:"batch_size,omitempty"`
```

Comment:
This is unrelated to the nil crash itself but it is to make the test deterministic: it will send the spans when the desired number of spans is batched in the test, so we can be sure that they arrive in a single batch. 

> updated the tests to include such cases and added a batch size config to the Zipkin exporter to ensure correctness of the test.

Reply:
I believe the proper way would be to add a Flush method to the exporter which wouldn't require time fiddling which as you know unfortunately goes haywire once we do `go test -race`.

Reply:
Yes, as we discussed in other review, Flush is preferable for this type of test, since I added the dedicated tests we can drop this added config since coverage stays the same.